<launch>
	<node name="autopilot_interface_node" pkg="autopilot_interface" type="autopilot_interface_node" output="screen">
		<param name="config_path" value="$(find autopilot_interface)/config/cessna.json" />
	</node>
	<node name="radio_comm_node" pkg="radio_comm" type="radio_comm_node" output="screen">
		<param name="config_path" value="$(find radio_comm)/config/tnt.json" />
	</node>
	<node name="ground_station_node" pkg="ground_station" type="ground_station_node" output="screen">
		<param name="layout_path" value="$(find ground_station)/config/layoutCessna.json" />
		<param name="resource_path" value="$(find ground_station)/resources/" />
		<param name="config_path" value="$(find ground_station)/config/cessna.json" />
	</node>
	<node name="camera_simulation_node" pkg="camera_simulation" type="camera_simulation_node" output="screen">
		<param name="base_img_path" type="str" value="/home/pure/devel/uavEE/src/camera_simulation/image.jpg" />
		<param name="output_path"  type="str" value="/home/pure/devel/uavEE/build/test_results/camera_results/output.avi" />
		<param name="convert_ratio" type="double" value="1" />
		<param name="udp_radius" type="double" value="100" />
		<param name="udp_origin_x" type="double" value="-850" />
		<param name="udp_origin_y" type="double" value="500" />
	</node>
</launch>
